README.md

```diff
@@ -174,8 +174,8 @@ APIs are subject to change without warning until v0.1.0.
 
 torchgpipe project is developed by [Heungsub Lee][], [Myungryong Jeong][], and
 [Chiheon Kim][] at [Kakao Brain][], with [Sungbin Lim][], [Ildoo Kim][],
-[Woonhyuk Baek][], and [Boogeon Yoon][]'s help. It is distributed under [Apache
-License 2.0](LICENSE).
+[Woonhyuk Baek][], and [Boogeon Yoon][]'s help. It is distributed under [BSD-3-Clause
+License](LICENSE).
```

Comment:
"BSD-3-Clause" is the short identifier of this license. For humans, we have a better notation "The 3-Clause BSD License". I think it looks better in English. (Reference: https://opensource.org/licenses/BSD-3-Clause)

```suggestion
[Woonhyuk Baek][], and [Boogeon Yoon][]'s help. It is distributed under [the 3-clause
BSD license](LICENSE).
```